meilisearch-schema/src/schema.rs

```diff
@@ -158,7 +158,7 @@ impl Schema {
             OptionAll::Some(ref v) => v.iter().cloned().collect(),
             OptionAll::All => {
                 let fields = self
-                    .indexed_map
+                    .fields_position
                     .iter()
                     .map(|(&f, _)| f)
```

Comment:
Is it just `fields_position.keys().cloned().collect()` here?

Reply:
yes that's right, fixed